### PR TITLE
Fusion and Inventor glTF exporters

### PR DESCRIPTION
**Pull request by liamjwang**

### CODE REVIEW

File: exporter/SynthesisInventorGltfExporter/README.md [hunk 1]
```diff
@@ -40,3 +40,6 @@ Changes to the exporter code aren't taking effect / The exporter add-in won't st
 - Delete all of the files at `C:\Program Files\Autodesk\Synthesis\SynthesisInventorGltfExporter`
 - Clean and build the solution, and ensure the built files are getting copied to `C:\Program Files\Autodesk\Synthesis\SynthesisInventorGltfExporter`
 - Ensure the `Autodesk.SynthesisInventorGltfExporter.Inventor.addin` file is getting copied to `%AppData%\Autodesk\ApplicationPlugins\SynthesisInventorGltfExporter`
+
+Visual Studio shows a "System.NullReferenceException: 'Object variable or With block variable not set'" and/or "Microsoft.VisualBasic.pdb not loaded":
+- Cause is currently unknown, but the exporter seems to work anyways if you uncheck "Break when this exception type is thrown" on the exception window and press continue.
```
Comment (liamjwang) [inline exporter/SynthesisInventorGltfExporter/README.md:45]: @matthew17754 Could you see if this fix works as an alternative to spamming the continue button?

Comment (matthew17754) [reply exporter/SynthesisInventorGltfExporter/README.md:45]: @liamjwang Yeah, that's a much better solution.

**Comment by matthew17754** [comment]
@victoryforphil Can you review the Fusion360 implementation?

**Comment by victoryforphil** [comment]
lgtm 👍

**Pull request merged**